Terminal.Gui/ConsoleDrivers/CursesDriver/CursesDriver.cs

```diff
@@ -30,7 +30,7 @@ internal class CursesDriver : ConsoleDriver {
 		IClipboard clipboard;
 		int [,,] contents;
 
-		internal override int [,,] Contents => contents;
+		public override int [,,] Contents => contents;
```

Comment:
Well, when the `Contents` was only needed basically for unit tests, this was irrelevant. But when the `Border` class was created, it was needed for to write the `3D Offset`, then it passed to be mandatory for all drivers and thus it must be `abstract` or at least `virtual`. The refactor indeed suggest to convert it to a method, but `dotnet` has many only `getter` properties or with `private/internal setters` and doesn't represent any break to design pattern. Since it's already used as is, I suggest maintain as `getter` property, if you don't mind.